src/main/resources/db/migration/V1_15__insert_court_type_to_service_mapping.sql

```diff
@@ -0,0 +1,87 @@
+insert into court_type_service_assoc(
```

Comment:
@AdilOoze  this insert script was part of ticket ?

Reply:
@shreedhar-hmcts as per @kavithagade12 we needed to add any files from LRD API that were missing in LRD Load to keep them in sync - hence why there are two extra insert scripts in this PR

Reply:
Sorry @AdilOoze Is the insert and update statements sql should be in capitals for readability otherwise kindly ignore it Thanks :)

Reply:
@sahitya-desireddy please note that there are old scripts that are being added from LRD API to Load - these are not new

Reply:
Thank you @AdilOoze  👍

Reply:
@sahitya-desireddy - changes made here: https://github.com/hmcts/rd-location-ref-data-load/pull/245/commits/02cf8c20671fa945687897b7286454fb4b5e94d2

Reply:
ok , please keep this communication in JIRA so that there will be log why we are adding this extra script. Also is this script is required in production env?

Reply:
@shreedhar-hmcts sure will keep this in JIRA - I think @kavithagade12 will be able to answer your question best as it was her decision

Reply:
For some reason data load and api projects are not in sync. I want both data load and api projects to be with the same versioning.

Reply:
@shreedhar-hmcts - these changes are in load. I think these changes will  available only in PR as it flyway is disabled from AAT

Reply:
ok thats fine